src/index.html

```diff
@@ -7,7 +7,58 @@
     <link rel="stylesheet" href="./index.scss">
   </head>
   <body>
-    <h1>Hello, email client!</h1>
+    <div class="wrapper">
+      <div class="email-client">
+        <div class="email-list-container">
```

Comment:
In the interest of HTML semantics, can we make this an `aside` tag?

Reply:
👍